### PR TITLE
Feature issues5and8

### PR DESCRIPTION
**Pull request by mpaulosky**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    /Taml.NET                          -- DotNet Parser
+    /Test.Taml.NET                     -- Unit tests
+  /python
+    /src
+      /pytaml                          -- Python Parser
+    /tests                             -- Unit tests
+
+```
+
+All official versions of the project are built and delivered with GitHub Actions and linked in the main README.md and [releases tab in GitHub](https://github.com/FritzAndFriends/BlazorWebFormsComponents/releases).
```
Comment (csharpfritz) [inline CONTRIBUTING.md]: Let's update this URL to this repository

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
+# Contributing to BlazorWebFormsComponents
```
Comment (csharpfritz) [inline CONTRIBUTING.md]: Update to TAML

Comment (mpaulosky) [reply CONTRIBUTING.md]: Completed this change.

File: CONTRIBUTING.md (hunk 1 above)
Comment (mpaulosky) [reply CONTRIBUTING.md]: Corrected this link

**Comment by csharpfritz** [comment]
Thank you for adding these two important docs to our project

**Pull request merged**